scss/_alert.scss

```diff
@@ -50,6 +51,7 @@
 // Provide class for links that match alerts
 .alert-link {
   font-weight: var(--#{$prefix}alert-link-font-weight); // Boosted mod
+  color: var(--#{$prefix}alert-link-color);
```

Comment:
Need to integrate `:hover, :active and/or :disabled .disabled` states ?

Reply:
I don't understand this question. There weren't in Bootstrap nor in Boosted if I'm right.

Reply:
Yeah I see. I'll need to remove `color` rule from `.alert-link` in Boosted an probably also drop `--bs-alert-link-color` that will become useless.

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/9d43843b39cea7b380a09f29c5297c8589ea4f66